src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -0,0 +1,93 @@
+package com.scottlogic;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.List;
+
+public class AuthorPostSorterTest {
+
+    AuthorPostSorter sortByAuthor = new AuthorPostSorter();
+    private UserPost createUserPost (String name ) {
+
+        return new UserPost(name,
+                OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+                "Hello World!", 2);
+    }
+
+
+
+
+
+    @Test
+    public void authorSorter_mixedValidNames_correctListReturnedAsc() {
```

Comment:
The first part of the test name is usually the name of the method being tested, i.e. `sort`. I also think the 'ascending' part would be better placed in the middle section, which describes the setup. Similarly for subsequent tests.